src/shader.cpp

```diff
@@ -79,8 +79,9 @@ void Context::glLinkProgram(GLuint program)
         return;
     }
 
-    auto& programr = program_it->second;
+    auto& programInfo = program_it->second;
```

Comment:
`program_info`  for consistent naming :)

Reply:
I thought I fixed this :0

Reply:
Alright, fixed for real.